tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj

```diff
@@ -265,7 +300,7 @@
     <When Condition="($(TargetFrameworkIdentifier) == '.NETStandard' And ($(TargetFrameworkVersion) == 'v1.3' Or $(TargetFrameworkVersion) == 'v1.4' Or $(TargetFrameworkVersion) == 'v1.5' Or $(TargetFrameworkVersion) == 'v1.6' Or $(TargetFrameworkVersion) == 'v2.0')) Or ($(TargetFrameworkIdentifier) == '.NETCoreApp' And ($(TargetFrameworkVersion) == 'v1.0' Or $(TargetFrameworkVersion) == 'v1.1' Or $(TargetFrameworkVersion) == 'v2.0'))">
       <ItemGroup>
         <Reference Include="System.Collections.NonGeneric">
-          <HintPath>..\..\packages\test\System.Collections.NonGeneric\ref\netstandard1.3\System.Collections.NonGeneric.dll</HintPath>
+          <HintPath>..\..\packages\test\System.Collections.NonGeneric.4.3.0\ref\netstandard1.3\System.Collections.NonGeneric.dll</HintPath>
```

Comment:
I was missing the group with paket why.

It DOES come from NUnit:

![image](https://user-images.githubusercontent.com/4236651/28943735-280318c4-78a0-11e7-91f5-775dfd5c6d39.png)